tutorials/index.md

```diff
@@ -11,10 +11,7 @@ children:
 Here are a collection of Docker tutorials that showcase how to use the Brooklyn entities provided by Clocker.
 
 ### Docker Swarm
-Clocker can deploy and manage Swarm clusters, as well as applications onto those clusters. Here are tutorials to help you achieve this:
-
-* [Swarm Cluster Tutorial](swarm-cluster.html): Deploying a production ready Swarm cluster.
-* [Swarm Location Tutorial](swarm-location.html): Deploying apps to a Swarm cluster.
+Clocker can deploy and manage Swarm clusters on different clouds. [Follow this tutorial](swarm-cluster.html) to learn how to.
 
 ### Kubernetes cluster
-Clocker is able to deploy a Kubernetes cluster on different clouds. [Follow this tutorial](kubernetes-cluster.html) to learn how to.
\ No newline at end of file
+Clocker is able to deploy a Kubernetes cluster on different clouds. [Follow this tutorial](kubernetes-cluster.html) to learn how to.
```

Comment:
Looks good but now we have the same sentence for Kubernetes and Swarm. I would rewrite the file entirely like this:

```markdown
## Overview
Here are a collection of tutorials that showcase how to use the blueprints provided by Clocker:
* [Deploy Docker Swarm](swarm-cluster.html)
* [Deploy Kubernetes cluster](kubernetes-cluster.html)
```

@drigodwin Thoughts?